config/schema_descriptions.yml

```diff
@@ -63,9 +63,9 @@ reporting_patient_states:
     quarters_since_registration: "Number of quarters since registration. If a patient was registered on 31st Dec, it would be 1 quarter since registration on 1st Jan."
     months_since_visit: "Number of months since the patient's last visit. If a patient visited on 31st Jan, it would be 1 month since the visit on 1st Feb."
     quarters_since_visit: "Number of quarters since the patient's last visit. If a patient visited on 31st Jan, it would be 1 quarter since the visit on 1st Jan."
-    months_since_bp: "Number of months since the patient's last BP measurement. If a patient had a BP reading on 31st Jan, it would be 1 month since BP on 1st Feb."
-    quarters_since_bp: "Number of quarters since the patient's last BP measurement. If a patient had a BP reading on 31st Jan, it would be 1 quarter since BP on 1st Jan."
-    last_bp_state: "The state of the last BP measured: controlled, uncontrolled, or unkown"
+    months_since_bp: "Number of months since the patient's last BP recording. If a patient had a BP reading on 31st Jan, it would be 1 month since BP on 1st Feb."
+    quarters_since_bp: "Number of quarters since the patient's last BP recording. If a patient had a BP reading on 31st Jan, it would be 1 quarter since BP on 1st Jan."
+    last_bp_state: "The state of the last BP recorded: controlled, uncontrolled, or unknown"
```

Comment:
Interesting. Should we add the threshold for control (140/90) to this description? Or leave that decoupled from these field descriptions?

Reply:
Yeah, the definition and comments might diverge if we add it in both places.